test/unit/node_stream_spec.js

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* globals __non_webpack_require__ */
+
+import { isNodeJS } from '../../src/shared/util';
+import { PDFNodeStream } from '../../src/display/node_stream';
+
+describe('node_stream', function() {
+  let pdf1 =
+    'http://mozilla.github.io/pdf.js/web/compressed.tracemonkey-pldi-09.pdf';
```

Comment:
> Why does this test have a special case !isNode() if the API that you are testing is only supposed to work on Node? 

Sorry, removed with new commit, as we don't that any more.

> let's also add assert(isNode()); just in case somebody will decide to add it.

Done. Thanks.